tests/integration/cli/test_cli.py

```diff
@@ -8,44 +11,32 @@
 
 import pytest
 import torch
-from tests.legacy.helpers.dataset import get_dataset_path
 
 from anomalib.utils.cli import AnomalibCLI
 
 
 class TestCLI:
     """Do sanity check on all models."""
 
-    def test_fit(self, model_name: str, dataset_root: Path, project_path: Path) -> None:
+    def test_fit(self, dataset_path: Path, project_path: Path) -> None:
         """Test fit CLI.
 
         Args:
-            model_name (str): Name of the model to train.
-            dataset_root (str): Root of the synthetic/original dataset.
-            project_path (str): Path to temporary project folder.
+            dataset_path (Path): Root of the synthetic/original dataset.
+            project_path (Path): Path to temporary project folder.
         """
-        data_class = "anomalib.data.MVTec"
-        category = "shapes"
-        if model_name == "ai_vad":
-            # TODO(ashwinva): use dummy dataset path when it supports all the data formats
-            # CVS-109972
-            # aivad expects UCSD dataset
-            data_class = "anomalib.data.UCSDped"
-            dataset_root = get_dataset_path(dataset="ucsd")
-            category = "UCSDped2"
-
         # batch size of 8 is taken so that the lr computation for efficient_ad does not return 0 when max_epochs is 1
         AnomalibCLI(
             args=[
                 "fit",
-                "-c",
-                f"src/configs/model/{model_name}.yaml",
+                "--model",
+                "Padim",
```

Comment:
are we gonna always use `Padim` or randomly choose it in the future?

Reply:
Yeah let's do that. The only issue is that AIVad does not work with MVTec but we can select from the rest.

Reply:
we could split image and video datasets/models in the future maybe?